backstop/tests/button.html

```diff
@@ -1172,12 +1176,12 @@ <h3>Borderless</h3>
     <h3>Split menu</h3>
     <section id="demo-split-menu">
       <span class="iui-button-split-menu">
-        <div>
+        <div class="iui-button-group-item">
```

Comment:
Looks weird that we are using button group item class here but container is split button and there is nothing else from button group.

Reply:
![image](https://user-images.githubusercontent.com/9084735/152220365-934347e0-372d-41bd-bc56-e63e6ae00a19.png)

Reply:
This class is removed so it's no longer a concern